parrot_cms_setup

```diff
@@ -267,7 +273,7 @@ fi
 storage_xml="$OSG_APP/cmssoft/cms/SITECONF/local/PhEDEx/storage.xml"
 if ! [ -e "$storage_xml" ]; then
     # At non-CMS sites, we may encounter an available CVMFS mount that is not configured.
-    if [ "X$VO_CMS_SW_DIR" == "" ]; then
+    if [ -z "$VO_CMS_SW_DIR" ]; then
         VO_CMS_SW_DIR=/cvmfs/cms.cern.ch
     fi
     site_config_xml="$VO_CMS_SW_DIR/SITECONF/local/PhEDEx/storage.xml"
```

Comment:
This line is also wrong, `site_config_xml` ==> `storage_xml`. This might a copy & paste issue from the above checks.